src/style.css

```diff
@@ -0,0 +1,123 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
Don't assign box-sizing to every element by hand, either do `* {
  box-sizing: border-box;
}` or create a separate file `reset.css` import there [some css reset like this one](https://meyerweb.com/eric/tools/css/reset/) and add that rule there

Reply:
Also it would be better to use flex on this whole block, and style accordingly